identity/src/main/java/com/android/identity/mdoc/origininfo/OriginInfoWebsite.java

```diff
@@ -26,43 +27,45 @@
 import co.nstant.in.cbor.model.Map;
 import com.android.identity.util.Logger;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
 public class OriginInfoWebsite extends OriginInfo {
```

Comment:
Since 18013-7 Consultation Draft defines two different types (1 and 2), looks like it would be better to have to distinct classes (`OriginInfoReferrerUrl` and `OriginInfoBaseUrl`) instead of `OriginInfoWebsite`.